### ISSUE TITLE
Question: customize CodeQL config to improve false positives?

### ISSUE THREAD

**Issue by pushfoo** [pyglet/pyglet#982]
## tl;dr

@Square789 and I have both had CodeQL flag lines in our PRs which were one or more of the following:

1. Pre-existing problems either near or overlapping with parts of the project we replaced
2. Useful to keep, such as commenting out logic which would be nice to have an ignore signal for

## Potential approaches

The `queries` section in the [`codeql.yaml`](https://github.com/pyglet/pyglet/blob/master/.github/workflows/codeql.yml) file should allow us to tune this.
https://github.com/pyglet/pyglet/blob/9d353098f7d956e38be17184bd70dcb5fdbbd525/.github/workflows/codeql.yml#L25-L41

For additional context, see:

* https://github.com/pyglet/pyglet/pull/973
* https://github.com/pyglet/pyglet/pull/975
